### DISCUSSION TITLE
Help Static files does not updating on production

### DISCUSSION THREAD

**Discussion by HelgiBergmann** [inventree/InvenTree#2550] [Q&A] [answered]
What can go wrong. I push my changes in production, then run update command, and nothing change. I even delete static folder, then run invoke update BUT get older static files again. Don't really know why, but docker does not update files in container.

**Comment by HelgiBergmann** [comment, marked as answer]
I have to Rebuild containers with no-cache option all the time then deploy to production, may be there is better option. If so, please tell.

**Reply by matmair** [reply]
@HelgiBergmann are you sure you run the invoke-command in the right container and path? It can get confusing with multiple environments. You could also try only running 'inv static'. In general, there shouldn't be that many updates that require static collection so I would recommend just running the command.

**Reply by matmair** [reply]
@HelgiBergmann did this help?
